system-contracts/scripts/preprocess-bootloader.ts

```diff
@@ -132,15 +132,15 @@ const params = {
   FORBID_ZERO_GAS_PER_PUBDATA: 1,
   SYSTEM_CONTEXT_EXPECTED_CODE_HASH: getSystemContextExpectedHash(),
   UPGRADE_SYSTEM_CONTEXT_CALLDATA: upgradeSystemContextCalldata(),
-  // One of "worst case" scenarios for the number of state diffs in a batch is when 120kb of pubdata is spent
-  // on repeated writes, that are all zeroed out. In this case, the number of diffs is 120k / 5 = 24k. This means that they will have
-  // accoomdate 6528000 bytes of calldata for the uncompressed state diffs. Adding 120k on top leaves us with
-  // roughly 6650000 bytes needed for calldata. 207813 slots are needed to accomodate this amount of data.
-  // We round up to 208000 slots just in case.
+  // One of "worst case" scenarios for the number of state diffs in a batch is when 240kb of pubdata is spent
+  // on repeated writes, that are all zeroed out. In this case, the number of diffs is 240k / 5 = 48k. This means that they will have
+  // accoomdate 13056000 bytes of calldata for the uncompressed state diffs. Adding 120k on top leaves us with
+  // roughly 13176000 bytes needed for calldata. 411750 slots are needed to accomodate this amount of data.
+  // We round up to 411900 slots just in case.
   //
   // In theory though much more calldata could be used (if for instance 1 byte is used for enum index). It is the responsibility of the
   // operator to ensure that it can form the correct calldata for the L1Messenger.
-  OPERATOR_PROVIDED_L1_MESSENGER_PUBDATA_SLOTS: 208000,
+  OPERATOR_PROVIDED_L1_MESSENGER_PUBDATA_SLOTS: 411900,
```

Comment:
Since the max size of pubdata per batch has increased to 240kb, you should also:

- Increase 2x the size of the user L2->L1 log tree.
- Increase 2x the space of the bootloader dedicated to bytecodes.
- While not mandatory per se, but it would be better if you increased the size of the bootloader memory to `30_000_000` (or maybe even more), so that the `BOOTLOADER_TX_ENCODING_SPACE` constant in the Rust could keep itself at the level of at least ~8Mb

Reply:
per @mm-zk comment, do we want to roll those into these changes or have them as a fast follow task?

Reply:
I think we should roll all those changes to allow for more comprehensive review/audit